docs/styleguide.md

```diff
@@ -62,6 +62,31 @@ export const DefaultStory = props => <ComponentName {...props} />;
 </Canvas>
 ```
 
+6. Add and link design files:
+
+It's nice to be able to access the relevant design files within Storybook itself. This is especially true when 
+we want to refer the design file for the relevant component in the future. We have [storybook-addon-designs](https://github.com/pocka/storybook-addon-designs) integrated to Storybook for this purpose. To link a design simply
+add a decorator to the `Meta` tag and the `design` parameter; 
```

Comment:
As suggested on https://pocka.github.io/storybook-addon-designs/?path=/docs/docs-quick-start--page, could we register the decorator globally so that we don't have to add this `withDesign` everywhere?

Reply:
Ops, I missed this part. Corrected it. 👍🏽